L/libnovas/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libnovas"
```

Comment:
I'd call the package 
```suggestion
name = "NOVAS"
```
that's the name of the upstream project.  Note that you'll have to rename the directory accordingly (and also L -> N)